src/alertsmanagement/azext_alertsmanagement/_params.py

```diff
@@ -0,0 +1,72 @@
+        c.argument('scope', nargs='+', help='List of ARM IDs (space-delimited) of the given scope type which will be the target of the given action rule.')
+        c.argument('severity', nargs='+', help='Filter alerts by severity. All filters should follow format "operator value1 value2 ... valueN". Operator is one of Equals, NotEquals, Contains and DoesNotContain.')
+        c.argument('monitor_service', nargs='+', help='Filter alerts by monitor service')
+        c.argument('monitor_condition', nargs='+', help='Filter alerts by monitor condition')
+        c.argument('target_resource_type', nargs='+', help='Filter alerts by target resource type')
+        c.argument('alert_rule', nargs='+', help='Filter alerts by alert rule name or ID')
+        c.argument('alert_description', nargs='+', help='Filter alerts by alert rule description')
+        c.argument('alert_context', nargs='+', help='Filter alerts by alert context (payload)')
+        c.argument('suppression_recurrence_type', arg_type=get_enum_type(SuppressionType), help='Specifies when the suppression should be applied')
+        c.argument('suppression_start_date', help='Start date for suppression. Format: MM/DD/YYYY')
+        c.argument('suppression_end_date', help='End date for suppression. Format: MM/DD/YYYY')
+        c.argument('suppression_start_time', help='Start time for suppression. Format: hh:mm:ss')
+        c.argument('suppression_end_time', help='End time for suppression. Format: hh:mm:ss')
+        c.argument('suppression_recurrence', nargs='+',
+                   help='List of recurrence pattern values, delimited by space. If recurrence type is Weekly, values '
+                   'should range from 0 to 6. 0 stands for Sunday, 1 stands for Monday, ..., 6 stands for Saturday. '
+                   'If recurrence type is Monthly, values should range from 1 to 31')
+
+    with self.argument_context('monitor action-rule update') as c:
+        c.argument('action_rule_name', action_rule_name)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+        c.argument('tags', tags_type)
+        c.argument('status', arg_type=get_enum_type(['Enabled', 'Disabled']), id_part=None, help='Indicates if the given action rule is enabled or disabled')
+
+    with self.argument_context('monitor action-rule delete') as c:
+        c.argument('action_rule_name', action_rule_name)
+
+    with self.argument_context('monitor action-rule show') as c:
+        c.argument('action_rule_name', action_rule_name)
+
+    with self.argument_context('monitor action-rule list') as c:
+        c.argument('target_resource_group', id_part=None, help='Filter by target resource group name. Default value is select all.')
+        c.argument('target_resource_type', id_part=None, help='Filter by target resource type. Default value is select all.')
+        c.argument('target_resource', id_part=None, help='Filter by target resource( which is full ARM ID) Default value is select all.')
```

Comment:
is `target_resource_group` paired with `target_resource`, again, I think CLI could get resource_group automatically instead of asking user input redundant information.  Given the long argument lists.

Reply:
They are separate target types. They are different filters. They are independent. Target can be resource or resource group.

Reply:
clear. thanks.